.github/DISCUSSION_TEMPLATE/docs.yml

```diff
@@ -0,0 +1,28 @@
+labels: ["kind/docs"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        #### Note
+        Feel free to create a docs report if something doesn't work as expected or is unclear in the documentation.
+        Please ensure that you're not creating a duplicate report by searching the [issues](https://github.com/aquasecurity/trivy/issues)/[discussions](https://github.com/aquasecurity/trivy/discussions) beforehand.
+        
+        Please also check [our contribution guidelines](https://aquasecurity.github.io/trivy/latest/community/contribute/discussion/).
+  - type: textarea
+    attributes:
+      label: Description
+      description: Briefly describe the what has been unclear in the existing documentation
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Link
+      description: Please provide a link to the current documentation or where you thought to find the information you were looking for
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Suggestions
+      description: What would you like to have added or changed in the documentation?
```

Comment:
aren't the two redundant?

"Briefly describe the what has been unclear in the existing documentation"
"What would you like to have added or changed in the documentation?"

Reply:
I want to encourage people to think about what would have made their experience more positive -- vs hearing what is wrong and then me trying to figure out how to improve it -- not saying that I would directly implement whatever people request. If you can think of better wording?

Reply:
you mean that these questions are mutually exclusive? I think most use cases for someone to create a docs issue are:
1. I found an error in the docs
2. I didn't find out how to do something
And I'm not sure how the two questions should be answered in these examples.

BTW if it makes sense to you it's fine, no need to justify it further